### DISCUSSION TITLE
Trouble installing tool

### DISCUSSION THREAD

**Discussion by axyrium** [svrooij/WinTuner#98] [Q&A]
I get all the pre-reqs installed, and successfully install the tool using
_dotnet tool install --global Svrooij.Winget-Intune.Cli_

but afterwards the commands aren't recognized: _"The term 'winget-intune' is not recognized as the name of a cmdlet, function, script file, or operable program"_
What am I doing wrong?

**Comment by svrooij** [comment]
it should install as `wintuner`, did I forget to change the docs somewhere?
And apart from that, you should check-out the powershell version, as stated [here](https://wintuner.app/docs/intro#wintuner-cli). The main development effort will go into the powershell thing. And I will eventually stop with the dotnet cli.
